mmaction/datasets/pipelines/loading.py

```diff
@@ -455,6 +455,50 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class SampleCharadesFrames(SampleFrames):
```

Comment:
docstring missing.

Pls briefly explain the difference of this class and the existing ones to help users understand why Charades needs a spearate sampling